buildSrc/src/main/kotlin/Libs.kt

```diff
@@ -0,0 +1,173 @@
+import kotlin.String
+
+/**
+ * Generated by https://github.com/jmfayard/buildSrcVersions
+ *
+ * Update this file with
+ *   `$ ./gradlew buildSrcVersions`
+ */
+object Libs {
+  /**
+   * http://developer.android.com/tools/extras/support-library.html
+   */
+  const val appcompat: String = "androidx.appcompat:appcompat:" + Versions.appcompat
+
+  /**
+   * https://developer.android.com/testing
+   */
+  const val espresso_core: String = "androidx.test.espresso:espresso-core:" + Versions.espresso_core
```

Comment:
I'm new to Kotlin Gradle DSL.  is it convention to use snakecase?  or just whatever you'd like?

Reply:
Not really sure what the convention is for Kotlin Gradle DSL, but I can't imagine it should stray too far from Kotlin conventions. In either case, this is auto-generated code that shouldn't be edited. All hand written code should strictly follow Kotlin coding conventions.